proposals/aip-1.md

```diff
@@ -0,0 +1,229 @@
+
+
+While the [Agorex Board of Directors] should quite liberally accept **Draft** status proposals subject to the criteria above, merging a proposal with a **Final** status such as **Final** or **Replaced** requires more review and rigor since, by merging a Final proposal The Board is agreeing to execute the proposed process on behalf of the community.
+
+Once the proposal is reviewed and approved by The Board on behalf of the community, the status will be changed to "Final" and it will be merged. By merging an **Accepted** proposal, the Board is explicitly acknowledging their responsibility and intent to execute the proposed process on behalf of the community.
+
+
+## Deferred, Rejected, and Replaced ##
+An AIP can also be assigned status **Deferred**. The AIP author or editor can assign the AIP this status when no progress is being made on the AIP. Once an AIP is deferred, the AIP editor can re-assign it to Draft status.
+
+An AIP can also be **Rejected**. Perhaps after all is said and done it was not a good idea. It is still important to have a record of this fact.
+
+AIPs can also be superseded by a different AIP, rendering the original obsolete. In this the superseeded Proposal should be marked as **Replaced** as part of the PR that the superseeding proposal is being proposed in.
+
+
+## Proposal Statuses ##
+The possible paths of the status of AIPs are as follows:
+
+  Draft -> Accepted -> Final
+  Draft -> Withdrawn
+  Draft -> Rejected
+  Draft -> Deferred
+  Final -> Replaced
+
+Unlike the EIP process, Informational and Process AIPs do not have the status "Active" and will have the status Final when they are accepted and the board is expected to execute the proposals on behalf of the community. 
+
+
+## Proposal Maintenance & Updates ##
+Informational and Process AIPs may be updated over time to reflect changes to processes and other details. The precise process followed in these cases will depend on the nature and purpose of the AIP being updated.
+
+# Proposal Structure: What belongs in a successful AIP? #
+
+Each AIP should have the following parts:
+
+- Preamble - [RFC 822] style headers containing metadata about the AIP, including the AIP number, a short descriptive title (limited to a maximum of 44 characters), the names, and optionally the contact info for each author, etc.
+
+- Simple Summary - "If you can’t explain it simply, you don’t understand it well enough." Provide a simplified and layman-accessible explanation of the AIP.
+
+- Abstract - a short (~200 word) description of the technical issue being addressed.
+
+- Copyright/public domain - Each AIP must either be explicitly labeled as placed in the public domain (see this AIP as an example).
+
+- Motivation - The motivation should motivated the proposal including a clear explanation of the problem that the AIP solves and why a current process or (lack thereof) is inadequate to address the problem. AIP submissions without sufficient motivation may be rejected outright. 
```

Comment:
Should be:

"The motivation should explain what motivated..."

Reply:
Fixed.